opr/functions.py

```diff
@@ -38,6 +39,34 @@ def basic_melting_temperature_calc(sequence):
     return melting_temperature
 
 
+def salt_adjusted_melting_temperature_calc(sequence, na_concentration=0.050):
+    """
+    Calculate the salt-adjusted melting temperature (Tm) of a DNA sequence.
+
+    :param sequence: Primer nucleotides sequence
+    :type sequence: str
+    :param na_concentration: Sodium ion concentration in moles (default 50 mM)
+    :type na_concentration: float
+    :return: Salt-adjusted melting temperature as float
+    """
+    a_count = sequence.count('A')
+    t_count = sequence.count('T')
+    c_count = sequence.count('C')
+    g_count = sequence.count('G')
+    seq_length = len(sequence)
+    salt_adjustment = 16.6 * (math.log10(na_concentration)) - 16.6 * (math.log10(0.050))
```

Comment:
Remove extra parenthesis to make it simpler to read.
`salt_adjustment = 16.6 * math.log10(na_concentration) - 16.6 * math.log10(0.050)`